### DISCUSSION TITLE
Official Documentation / Reference for Cutlass?

### DISCUSSION THREAD

**Discussion by RylanYancey** [NVIDIA/cutlass#572] [Q&A]
Hi, I'm fairly new to HPC and Machine learning, but I've written some Cuda Kernels and my own Machine Learning algorithm. I'm trying to learn how to do Matrix Multiplication on the GPU with Cutlass, but there does not seem to be any documentation or reference for it. All of the links on the Documentation page just go to Home, (except for quick start guide). Is there a reference for Cutlass where everything (or mostly everything) is documented?

if not, get on that this stuff is hard. 

Thanks! :D

**Comment by hwu36** [comment]
documents are here.  https://github.com/NVIDIA/cutlass/tree/master/media/docs

there are also many cutlass gtc talks: https://github.com/NVIDIA/cutlass/tree/master/media/docs

last, don't get scared of it.  it is just gemm and convolution, high school stuff.  in fact, many high school students use cutlass for their school projects.

**Reply by RylanYancey** [reply]
Thanks! :D
